src/app/pages/resources/dashboard/dashboard.jsx

```diff
@@ -82,23 +102,26 @@ function ChplDashboard() {
                     <Button
                       style={{ justifyContent: 'flex-start' }}
                       color="primary"
-                      onClick={() => handleCardChange('home')}
+                      onClick={() => handleReportChange(undefined)}
                     >
                       Dashboard
                     </Button>
-                    <Button
-                      style={{ justifyContent: 'flex-start' }}
-                      color="primary"
-                      onClick={() => handleCardChange('developerReport')}
-                    >
-                      Developer Report
-                    </Button>
+                    { reports.map((report) => (
+                      <Button
+                        key={`${report.reportId}-button`}
+                        style={{ justifyContent: 'flex-start' }}
+                        color="primary"
+                        onClick={() => handleReportChange(report.reportId)}
+                      >
+                        { report.text }
+                      </Button>
+                    ))}
                   </Box>
                 </CardContent>
               </Card>
             </Box>
             <Box width="100%">
-              {activeCard === 'home' && (
+              {activeReport == null && (
```

Comment:
```suggestion
              {!activeReport && (
```

Two things: 
1) `undefined !== null`, so I don't think this check works
2) In JS, `==` and `===` are different, but `===` is more strict, and what we should be using everywhere

That said, the easy answer is to just use the `!activeReport` check as in the code suggestion, and that would handle both `null` and `undefined`